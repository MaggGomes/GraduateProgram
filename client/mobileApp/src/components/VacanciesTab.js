import React, { Component } from 'react';
import {StyleSheet, Text, View} from 'react-native';

/* Vacancies Tab */
export default class VacanciesTab extends Component {
    render() {
        return (
            <View style={styles.container}>
                <Text style={styles.mainText}>Vacancies</Text>
            </View>
        );
    }
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        justifyContent: 'center',
        alignItems: 'center',
        backgroundColor: '#F5FCFF',
    },
    mainText: {
        fontSize: 20,
        textAlign: 'center',
        margin: 10
    }
});