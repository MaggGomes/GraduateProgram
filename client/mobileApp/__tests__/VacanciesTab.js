import React from 'react';
import renderer from 'react-test-renderer';
import VacanciesTab from '../src/components/VacanciesTab';

describe('VacanciesTab component', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<VacanciesTab />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});