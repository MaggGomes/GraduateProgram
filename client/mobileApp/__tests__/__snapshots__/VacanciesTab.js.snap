// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VacanciesTab component renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    Vacancies
  </Text>
</View>
`;
